utils/general.go

```diff
@@ -1,34 +1,42 @@
 package utils
 
 import (
+	"bytes"
+
+	"github.com/hashicorp/consul/api"
 	"github.com/spf13/viper"
-	_ "github.com/spf13/viper/remote"
 )
 
-// LoadConfiguration determines the app config store (consul or etcd) from the environment and loads full configuration.
+// LoadConfiguration determines the app config store (consul) from the environment and loads full configuration.
 //
 // Example env:
-//   APP_CONFIG_PROVIDER = consul
+//   APP_CONFIG_TOKEN = 0a47d149-a608-7b83-29c3-5b1f4d89e986
 //   APP_CONFIG_HOST = localhost:8500
-//   APP_CONFIG_PATH = ./my-application-config.json
```

Comment:
we defaulted to consul now so probably makes sense to name vars to reflect this to avoid confusion.